src/foam/u2/view/FObjectView.js

```diff
@@ -79,6 +79,12 @@ foam.CLASS({
         model in the 'of' property. The user can choose to create an instance
         of one of the models in this list.
       `
+    },
+    {
+      class: 'Boolean',
+      name: 'strategizerDisabled',
```

Comment:
Can we make this more positive and reverse the meaning? Like 'enableStrategizer' and then make the default true.